### PR TITLE
update jenkins doc for new secret->credential sync feature

### PR DESCRIPTION
**Pull request by gabemontero**
doc changes for https://trello.com/c/SGvNKyMM/1154-5-make-openshift-secrets-available-in-jenkins-as-credentials-jenkinsintegration

@openshift/sig-developer-experience @bmcelvee ptal

### CODE REVIEW

**Comment by gabemontero** [comment]
[(5) Make openshift secrets available in jenkins as credentials [jenkins_integration]](https://trello.com/c/SGvNKyMM/1154-5-make-openshift-secrets-available-in-jenkins-as-credentials-jenkinsintegration)

File: using_images/other_images/jenkins.adoc [hunk 1]
```diff
@@ -665,6 +665,39 @@ monitors the API server of {product-title} for updates to `BuildConfigs` and
 projects (when a `BuildConfig` is created) or starts jobs in the resulting
 projects (when a `Build` is started).
 
+Additionally, as noted xref:using-the-jenkins-kubernetes-plug-in-to-run-jobs[above],
+this plug-in can create `PodTemplate` configuration for the Kubernetes plug-in
```
Comment (bparees) [inline using_images/other_images/jenkins.adoc]: configurations

File: using_images/other_images/jenkins.adoc [hunk 2]
```diff
@@ -665,6 +665,39 @@ monitors the API server of {product-title} for updates to `BuildConfigs` and
...
+based on specifically cited `ImageStream`, `ImageStreamTag`, or `ConfigMap`
+objects defined in {product-title}.
+
+Lastly, this plug-in can now take `Secret` objects with a label key of
+`credential.sync.jenkins.openshift.io` and label value of `true` and construct
+Jenkins credetials which are placed in the default global domain within
```
Comment (bparees) [inline using_images/other_images/jenkins.adoc]: credentials

**Review by bparees — CHANGES_REQUESTED** [review]
couple nits and lgtm

**Comment by gabemontero** [comment]
spelling corrections pushed

**Comment by bparees** [comment]
lgtm

**Comment by gabemontero** [comment]
@bmcelvee - re: the card referenced above, this is 3.9 content

**Comment by bmcelvee** [comment]
Thanks @gabemontero! LGTM, I'm just going to make some minor edits in a follow-up PR.

**Comment by bmcelvee** [comment]
[rev_history]
|xref:../using_images/other_images/jenkins.adoc#using-images-other-images-jenkins[Jenkins]
|Added credential sync content to the xref:../using_images/other_images/jenkins.adoc#sync-plug-in[{product-title} Sync Plug-in] section.
%

**Pull request merged**